chapters/en/module2.md

```diff
@@ -0,0 +1,789 @@
+and submit it to validate if you were correct.**
+
+
+Alright, let's bring in a new dataset containing information on penguins. This data is available from [Kaggle](https://www.kaggle.com/parulpandey/palmer-archipelago-antarctica-penguin-data) and originally available from [Allison Horst's GitHub Repository](https://github.com/allisonhorst/palmerpenguins). 
+
+
+
+<codeblock id="penguins">
+
+</codeblock>
+
+
+We are interested in obtaining the number of penguins we collected of each species. 
+
+Tasks: 
+
+- Using a bar plot, visualize the count of penguins for each species in the data. 
+- Choose the appropriate rotation, given the axis labels. 
+- Sort your bars in an appropriate order. 
+- Make sure to give it a title and set the dimensions to a height of 350 and a width of 500.
+- Save your plot in an object named `penguin_bar`.
+
+<codeblock id="02_11">
+
+- Are you using `.mark_bar()`?
+- Are you assigning the plot to an object named`penguin_bar`?
+- Are you specifying `alt.Y('species', sort='x')` on the y-axis?
+- Are you specifying `count()` on the x-axis?
+- Are you giving the plot a title and proper dimensions using `.properties()`?
+
+</codeblock>
+
+**Question**      
+
+Which species is most occurring in the dataset?
+
+<choice id="1" >
+<opt text="Chinstrap">
+
+Are you looking at the right plot?
+
+</opt>
+
+<opt text="Gentoo" >
+
+Maybe try again.
+
+</opt>
+
+<opt text="Adelie" correct="true">
+
+Nice!
+
+</opt>
+
+</choice>
+
+</exercise>
+
+<exercise id="12" title="Making History with Histograms">
+
+
+**Instructions:**    
+Be patient when running a coding exercise for the first time, it can take a few minutes. 
+
+**When you see `____` in a coding exercise, replace it with what you assume to be the correct code. Run the code to see if you obtain the desired output
+and submit it to validate if you were correct.**
+
+
+Let's now see what the distribution of penguin flipper length looks like. 
+
+Tasks: 
+
+- Using bars, show the counts of the `flipper_length_mm` values. 
+- Make sure to give it a title and set the dimensions to a height of 350 and a width of 500.
+- Save your plot in an object named `penguin_hist`.
+
+<codeblock id="02_12">
+
+- Are you using `.mark_bar()`?
+- Are you assigning the plot to an object named`penguin_hist`?
+- Are you specifying `alt.X('flipper_length_mm', bin=True)` on the x-axis?
+- Are you specifying `count()` on the y-axis?
+- Are you giving the plot a title and proper dimensions using `.properties()`?
+
+</codeblock>
+
+</exercise>
+
+
+<exercise id="13" title="Creating Subplots via Faceting" type="slides,video">
+<slides source="module2/module2_13" shot="1" start="0:003" end="07:12"> </slides>
+</exercise>
+
+
+<exercise id="14" title="True or False: Facets">
+
+**True or False**       
+*If faceting, it's important to assign a color channel to the same column.*
```

Comment:
I find this question confusing... I need more context...

Reply:
Ok I've changed it to this: 

> **True or False**       
*If you faceting and assign a color channel to the same column, you will obtain subplots all in different colours*

> Answer:
> Setting a colour channel column to the same column as we are  facetting on,  will change each of the sub plots to different colours.

Still confusing? Should I come up with a new question?

Reply:
I think this is much better, it is not necessarily "important" to do this.